### PR TITLE
redux

### PR DESCRIPTION
**Pull request by Kinga-parasiewicz**
Hi there, 

I created searchSlice, searchSaga, rootSaga  and I added rootSaga.js and SearchSlice to store.  I started adding reducers to Search component but not finished because we need to  create file pageParameters and then  we can created arrow function on Submit and will be done. Please check and comment

### CODE REVIEW

File: src/features/Search/index.js [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+};
+
+
   return (
     <SearchWrapper>
       <StyledButtonSearch />
-      <Input placeholder={`Search for ...`} />
+      <Input placeholder={`Search for ... ${searchText}`} 
+      value={query}
+      open={isOpen}
```
Comment (izabelanowak) [inline src/features/Search/index.js]: Formating 😄

File: src/features/Search/index.js [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
+  const searchText = location.pathname.includes("movie") ? "movies" : "people";
+
+  useEffect(() => {
+    dispatch(setQuery(""));
+}, [location, dispatch])
+
+const onChange = (value) => {
+  dispatch(setQuery(value));
+  dispatch(setOpen(true));
+};
```
Comment (izabelanowak) [inline src/features/Search/index.js:21]: Formating 😄

**Review by izabelanowak — APPROVED** [review]
Very good job, Kinga. The code looks great, I have no comments apart from the formatting in Search / index.js 😄  correct the formatting and you can merge 😄

**Comment by izabelanowak** [comment]
Ok, I think we can close the pull request because we do not want to merge to main, but to develop, where these commits already are.

**Comment by krystian-kazmierczak** [comment]
Iza is right, you can close this PR.

**Pull request closed without merging**